src/views/Product.vue

```diff
@@ -64,25 +72,38 @@ export default defineComponent({
     IonPage,
     IonTitle,
     IonToolbar,
-    ProductDurationPopover
+    JobDetail
+  },
+  computed: {
+    ...mapGetters({
+      getJobStatus: 'job/getJobStatus'
+    }),
   },
   data() {
     return {
       jobEnums: JSON.parse(process.env?.VUE_APP_PRD_JOB_ENUMS as string) as any,
+      currentJobEnum: 'IMP_PRDTS',
+      title: 'Import products'
```

Comment:
Title should be internationalized

Reply:
Sir, it's internationalized on the JobDetail page, so not internationalizing it here.